infor-design/enterprise
# 3090 - Fix bug with addEvents [v4.23.x]

**Explain the _details_ for making this change. What existing problem does the pull request solve?**

Fixed a problem with the add API where it gave errors or added to the wrong date.

**Related github/jira issue (required)**:
Fixes #3090 

**Steps necessary to review your pull request (required)**:
- go to http://localhost:4000/components/calendar/example-index.html
- in the console type
```
$('.calendar').data('calendar').addEvent({
      subject: 'Test Event One',
      status: 'Pending',
      starts: '2019-12-02T15:00:00.000',
      ends: '2019-12-02T16:00:00.000',
      type: 'dto',
      isAllDay: 'false'
    })
```
- go to december and you should see this new date
- type
```
$('.calendar').data('calendar').addEvent({
      subject: 'Discretionary Time Off',
      shortSubject: 'DTO',
      comments: 'Personal time',
      location: 'Canada Office',
      status: 'Approved',
      starts: '2018-08-21T10:00:00.999',
      ends: '2018-08-21T14:00:00.999',
      type: 'dto',
      isAllDay: false
    })
```
- go to August 2018
- should say a DTO added on the 21st

**Included in this Pull Request**:
